src/Common/Utils/Net.ts

```diff
@@ -0,0 +1,11 @@
+﻿import * as express from "express";
+
+export module Net {
+    export function sendResponse(response: express.Response, result: any): void {
+        if (!result) {
+            response.status(404).send("The resource cannot be found")
```

Comment:
is this the preferred way to handle 404s in Express? This is essentially saying "send back a 404 whenever we haven't sent back a result", which could create problems in the future if you have APIs that are supposed to just return a status code without a response (or maybe if there's a HTTP 500 or HTTP 400 without a payload).

What happens if you don't do this, and somebody requests a path without a router? Does express also return a 404 by itself? If so, I'd recommend to just leave it at that.

Reply:
Reworked all error handling